src/test/ui/privacy/public-instead-of-pub.rs

```diff
@@ -0,0 +1,4 @@
+// Checks what happens when `public` is used instead of the correct, `pub`
+// edition:2018
+public struct MyStruct;
+//~^ ERROR 3:8: 3:14: expected one of `!` or `::`, found keyword `struct`
```

Comment:
```suggestion
//~^ ERROR expected one of `!` or `::`, found keyword `struct`
```
nit: you don't need to include the line/column numbers

Reply:
The test failed when I didn't include the line/column

Reply:
It definitely shouldn't - almost every test is written with error annotations that don't include the line/column numbers. What error are you seeing?

Reply:
Removing the line/column makes it fail [like this](https://user-images.githubusercontent.com/93856041/180987665-a91eb09c-2892-4359-85df-ec71a047419d.png)

Reply:
I think you just need to `--bless` the test again. The important part of that error is where it says "The actual fixed differed from the expected fixed" - that just means that the output it gets from running `rustfix` on the test is different from what it expects the output to be (this happens when you first add `// run-rustfix`, or when the test changes such that the output is genuinely different, such as if it the output had the comment with line numbers in it).

Reply:
https://github.com/rust-lang/rust/runs/7518469562?check_suite_focus=true

This also happens in CI, blessing the test again in a minute